config/mediaflux.yml

```diff
@@ -74,6 +74,7 @@ staging:
   api_password: <%= ENV["MEDIAFLUX_PASSWORD"] %>
   shared_files_location: <%= ENV["SHARED_FILES_MOUNT"] || '/mnt/nfs/tigerdata' %>
 development:
+  api_root: "tigerdata"
```

Comment:
I defined a new key `api_root` to use throughout the system now that we are using the `tigerdata.project.create` and it sticks all projects under `tigerdata`. At one point we might be able to remove the older keys (`api_root_ns`, `api_root_collection_name`, and so on) but didn't want to do this just yet.